utils/components-testkit/src/main/scala/pl/touk/nussknacker/engine/util/test/TestScenarioRunner.scala

```diff
@@ -45,8 +47,12 @@ trait TestScenarioRunner
 
 trait TestScenarioRunnerBuilder[R <: TestScenarioRunner, B <: TestScenarioRunnerBuilder[R, _]] {
 
+  def componentUseCase(testRuntimeMode: Boolean): ComponentUseCase = if (testRuntimeMode) TestRuntime else EngineRuntime
+
   def withExtraComponents(components: List[ComponentDefinition]): B
 
+  def inTestRuntimeMode(): B
```

Comment:
Let's remove parentheses - it has no side effect. `build` has parentheses because the name is in imperative form. We can discuss removing them if it is confusing